### ISSUE TITLE
the paper dataset

### ISSUE THREAD

**Issue by conquerSelf** [taoqi98/PP-Rec#2]
Dear the author, could you provide the paper dataset?? If you can, thank you so much!

**Comment by taoqi98** [comment]
Thanks a lot for your attention on our work. Due to privacy issues, data used in our paper cannot be released NOW. We will release the data ASAP. Fortunately, there is a public news recommendation dataset can be used to realize our method. You can try the MIND dataset with our code: https://msnews.github.io/index.html

**Comment by wzz2579421845** [comment]
> Dear the author, could you provide the paper dataset?? If you can, thank you so much!

Hello, have you received the data set about this paper？if you have one, can you send it to me, thank you very much！！

**Comment by Hannibal046** [comment]
Hi，
How to use your model with MIND dataset? Since there is no popularity information about the news.

**Comment by taoqi98** [comment]
The MIND dataset is based on logs of 1 million users in 6 weeks, where user logs in the first four weeks are used to construct historical user behavior, user logs in the first five days of the 5-th week are used to construct training dataset, and user logs in the last two days of the 5-th week are used to construct validation dataset. Note that the training and validation set provide raw news impressions including click, non-click, and impression time, we can calculate real-time click-through-rate for news in the 5-th week as the input of PP-Rec. Thus, in practice, the training set of MIND can be used to train PP-Rec, and the validation set of MIND can be used to verify the performance of PP-Rec. In addition, MIND does not provide news recency information, thus, we can only implement a variation of PP-Rec that only utilizes news content and CTR for popularity prediction.

**Comment by igor17400** [comment]
@taoqi98, first and foremost, thank you again for sharing your code, and I apologize if my question causes any inconvenience. 

**_Summary_**:  further understand how the CTR of **_historical_** clicked news is obtained for the MIND dataset.

In your paper, you detailed the "Popularity-aware user encoder" which calculates the popularity embedding for each user's historically clicked news article, based on the Click-Through Rate (CTR) for that specific news article. 

Regarding the implementation of this concept using the MIND dataset, which lacks direct CTR data for the "four weeks used to construct historical user behavior" I would like to know your recommendation on how to proceed on the first iteration of the user encoder. 

Should we start with a baseline CTR value of **_zero_** for these historial articles and adjust based on observed clicks during the dataset's training phase (5th week) since we don't have the number of clicks in each news article for the previous 4 weeks?
